### ISSUE TITLE
[Feature request] Add application to the F-Droid store

### ISSUE THREAD

**Issue by Serverfrog** [MenosGrante/Rekado#70]
It would be nice for a better dirstribution to have this App in the F-Droid Appstore.
A Positive benifit ist the easy install and update process the user will have.

I didn't dig into if the additional Requirements are met, but as far as is saw the Notably Part is met https://f-droid.org/en/docs/Inclusion_Policy/

Beside that, its also a bit secure because they build the APK from your repo, signed it and then distribute them. So the hassle to build it yourself and create a Release out of it could also be done by them (you still must upload it into the GitHub Releases but could use the F-Droid build APK )

Also: new users must not go to the hassle to Download the APK from GitHub if they already have F-Droid installed

**Comment by MenosGrante** [comment]
Hi, I planned to add applications there at the start, but then I was distracted by the "fascinating" process of fixing defects in the initial versions. But now that the application is stable enough, I thought it was time to add it.

So, in the future, it will certainly appear there, but first, you need to properly prepare the repository.

**Comment by MenosGrante** [comment]
Started the inclusion process: https://gitlab.com/fdroid/rfp/-/issues/1339

**Comment by MenosGrante** [comment]
Since Rekado uses my own library for interface customization and I would not like to make it open-source, so probably F-Droid inclusion is not possible. I have plans to create some mechanism for auto-updating of application from the inside application.

**Comment by Serverfrog** [comment]
@MenosGrante, if i understand correctly, this Library (penza.aar) is just for UI Customizations?

**Comment by MenosGrante** [comment]
@Serverfrog, yes, it is. I also have a variant to simply remove it from the application and create a simple mechanism for white-dark theme changing, without additional themes and accent color customization.

**Comment by nfitzen** [comment]
@MenosGrante is there any reason you couldn't make a separate fully free build, for inclusion in F-Droid? If you made a request using that variant, they'll most likely approve it.

**Comment by Magicrafter13** [comment]
> Since Rekado uses my own library for interface customization and I would not like to make it open-source, so probably F-Droid inclusion is not possible. I have plans to create some mechanism for auto-updating of application from the inside application.

Why don't you want your library to be libre?

**Comment by MenosGrante** [comment]
Actually, on the latest updates I have removed all closed-source code from it and now it should be fine to apply to F-Droid, but I am tight on time right now, so I am not sure when I can start the process.
